opentech/apply/funds/models.py

```diff
@@ -537,28 +560,24 @@ class ApplicationSubmission(WorkflowHelpers, BaseStreamForm, AbstractFormSubmiss
     search_data = models.TextField()
 
     # Workflow inherited from WorkflowHelpers
-    status = models.CharField(max_length=254)
+    status = FSMField(default=INITAL_STATE, protected=True)
 
     # Meta: used for migration purposes only
     drupal_id = models.IntegerField(null=True, blank=True, editable=False)
 
     objects = ApplicationSubmissionQueryset.as_manager()
 
-    @property
-    def status_name(self):
-        return self.phase.name
-
     @property
     def stage(self):
         return self.phase.stage
```

Comment:
getting `'NoneType' object has no attribute 'stage'` in https://github.com/OpenTechFund/opentech.fund/blob/6b7c63797916f9216bfe1f385499b94d51715e58/opentech/apply/funds/templates/funds/tables/column_reviews.html#L6